multisig/proposals/arbitrumone/vip-007/index.ts

```diff
@@ -0,0 +1,136 @@
+      params: [
+        PRIME_LIQUIDITY_PROVIDER,
+        "setMaxTokensDistributionSpeed(address[],uint256[])",
+        arbitrumone.NORMAL_TIMELOCK,
+      ],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxLoopsLimit(uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateAlpha(uint128,uint128)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateMultipliers(address,uint256,uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setStakedAt(address[],uint256[])", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "addMarket(address,address,uint256,uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setLimit(uint256,uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setMaxLoopsLimit(uint256)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "issue(bool,address[])", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "burn(address)", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "togglePause()", arbitrumone.NORMAL_TIMELOCK],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: XVS_VAULT_PROXY,
+      signature: "setPrimeToken(address,address,uint256)",
+      params: [PRIME, XVS, PRIME_POOL_ID],
+    },
+    {
+      target: PRIME,
+      signature: "setLimit(uint256,uint256)",
+      params: [
+        0, // irrevocable
+        0, // revocable
+      ],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "pauseFundsTransfer()",
+      params: [],
+    },
```

Comment:
We should resume the XVSVault at the end, as it was made at [007 on Ethereum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/ethereum/vip-007/index.ts#L117-L121)

`
    {
      target: XVS_VAULT_PROXY,
      signature: "resume()",
      params: [],
    },
`

Reply:
Done https://github.com/VenusProtocol/vips/pull/264/commits/56a49e0cf7511dda2a4edc9eae6eb88b17d76a19